### PR TITLE
Fix #201 : Fixed issue with oppiabot assigning undefined users

### LINKED ISSUE #201: Bot is asking @undefined user to add changelog label to the PR
OppiaBot is leaving is asking undefined users for adding a changelog label.

`Hi @undefined, @undefined -- could one of you please add the appropriate changelog label to this pull request? Thanks!`

Comment link: https://github.com/oppia/oppia/pull/11957#issuecomment-779131071

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation

Fixes #201: Fixed issue with oppiabot asking undefined users to add changelog label.
This issue usually occurred when there are no reviewers for a pull request (usually a draft PR). 

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: lib/checkPullRequestLabels.js [hunk 1]
```diff
@@ -215,15 +216,6 @@ module.exports.checkChangelogLabel = async function (context) {
...
   if (!isCollaborator) {
-    // Sleep for 5 seconds and refetch the pull request so it
-    // contains all the requested reviewers.
-    await sleep(5000);
-    console.log('Finished sleeping');
-    const pullRequestResponse = await context.github.pulls.get(
-      context.repo({ pull_number: pullRequestNumber })
-    );
-    pullRequest = pullRequestResponse.data;
-
```
Comment (vojtechjelinek) [inline lib/checkPullRequestLabels.js:226]: Why is this removed?

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

File: lib/checkPullRequestLabels.js (hunk 1 above)
Comment (jameesjohn) [reply lib/checkPullRequestLabels.js:226]: It is no longer required as this didn't fix the issue before.

File: lib/checkPullRequestLabels.js [hunk 2]
```diff
@@ -241,7 +233,7 @@ module.exports.checkChangelogLabel = async function (context) {
         reviewers[0] +
         ', could you please add the appropriate changelog label to this ' +
         'pull request? Thanks!';
-    } else {
+    } else if (reviewers.length > 1) {
```
[the remark this answers is not in the document]
Comment (jameesjohn) [reply lib/checkPullRequestLabels.js:236]: Yes. We don't need to ping all the reviewers to add the changelog label. 2 should be enough.

**Comment by vojtechjelinek** [comment]
@jameesjohn Some comments are not addressed. (https://github.com/oppia/oppiabot/pull/260#discussion_r624521591)

File: lib/checkPullRequestLabels.js [hunk 3]
```diff
@@ -250,6 +242,23 @@ module.exports.checkChangelogLabel = async function (context) {
...
+      console.log({pullRequest});
+      if (pullRequest.draft) {
+        // Do nothing since no review is required here.
+        return;
+      } else {
+        // No reviewers in the PR due to some other reason.
+        // Ping the welfare team lead
+        commentBody =
+          'Hi @' +
+          teamLeads.welfareTeam +
```
[the remark this answers is not in the document]
Comment (jameesjohn) [reply lib/checkPullRequestLabels.js]: Done.

**Review by vojtechjelinek — APPROVED** [review]
LGTM!
